src/frontend/src/pages/WorkPackageDetailPage/ActivateWorkPackageModalContainer/ActivateWorkPackageModal.tsx

```diff
@@ -39,25 +40,35 @@ interface ActivateWorkPackageModalProps {
 }
 
 const schema = yup.object().shape({
-  startDate: yup.date().required(),
+  startDate: yup
+    .date()
+    .required('Start Date is required')
+    .test('start-date-valid', 'start date is not valid', startDateTester),
   confirmDetails: yup.boolean().required()
 });
 
-const defaultValues: FormInput = {
-  startDate: new Date().toLocaleDateString(),
-  confirmDetails: false
-};
-
 const ActivateWorkPackageModal: React.FC<ActivateWorkPackageModalProps> = ({
   allUsers,
   wbsNum,
   modalShow,
   onHide,
   onSubmit
 }) => {
+  const startDate = new Date();
+  const today = startDate.getDay();
+  if (today !== 1) {
+    const daysUntilNextMonday = (7 - today + 1) % 7;
+    startDate.setDate(startDate.getDate() + daysUntilNextMonday);
+  }
+
+  const defaultValues: FormInput = {
+    startDate: startDate,
+    confirmDetails: false
+  };
+
   const { reset, handleSubmit, control } = useForm<FormInput>({
     resolver: yupResolver(schema),
-    defaultValues
+    defaultValues: defaultValues
```

Comment:
you can shorten this to just defaultValues, typescript knows to match up the attribute name and the const you've declared in this file